frontend/src/email/EmailForm.jsx

```diff
@@ -262,135 +275,150 @@ class EmailForm extends Component {
-          <div className="row">
-            <span className="col-md-6 hc-text">Service Client credentials</span><span
-              className="col-md-6 hc-ind"><span id="credentialsInd"
-                className={credentialsIndClass}></span></span>
-          </div>
-          <div className="row">
-            <span
-              className="col-md-6 hc-text">Service Client has access to Common Messaging API</span><span
-              className="col-md-6 hc-ind"><span id="apiAccessInd" className={apiAccessIndClass}></span></span>
+      <div className="col-md-8 offset-md-2 order-md-1">
+
+        <div className="text-center" style={displayBusy}>
+          <div className="spinner-grow text-primary" role="status">
+            <span className="sr-only">Loading...</span>
           </div>
-          <div className="row">
-            <span className="col-md-6 hc-text">Service Client can send message</span><span
-              className="col-md-6 hc-ind"><span id="createMsgInd" className={createMsgIndClass}></span></span>
+          <div className="spinner-grow text-primary" role="status">
+            <span className="sr-only">Loading...</span>
           </div>
-          <div className="row">
-            <span className="col-md-6 hc-text">Common Messaging API available</span><span
-              className="col-md-6 hc-ind"><span id="healthCheckInd"
-                className={healthCheckIndClass}></span></span>
+          <div className="spinner-grow text-primary" role="status">
+            <span className="sr-only">Loading...</span>
           </div>
         </div>
 
-        <hr className="mb-4"/>
-        <form id="emailForm" noValidate style={emailFormDisplay} onSubmit={this.formSubmit}
-          className={wasValidated ? 'was-validated' : ''}>
-          <h4 className="mb-3">Email</h4>
-          <div className="mb-3">
-            <label htmlFor="sender">Sender</label>
-            <input type="text" className="form-control" name="sender"
-              readOnly required value={this.state.form.sender}/>
-            <div className="invalid-feedback">
-              Email sender is required.
+        <div style={displayNotBusy}>
+          <div className="alert alert-success" style={successDisplay}>
+            {this.state.info}
+          </div>
+          <div className="alert alert-danger" style={errorDisplay}>
+            {this.state.error}
+          </div>
+          <h4 className="mb-3">Service Client</h4>
+          <div id="healthCheck">
+            <hr className="mb-4"/>
+            <div className="row">
+              <span className="col-sm-10 hc-text">Service Client credentials</span><span
+                className="col-sm-2"><span id="credentialsInd"
+                  className={credentialsIndClass}></span></span>
+            </div>
+            <div className="row">
+              <span
+                className="col-sm-10 hc-text">Service Client has access to Common Messaging API</span><span
```

Comment:
Split the spans to multiple lines maybe for these rows.

Reply:
Row/Col pairs are usually both divs as well. I've never actually checked to see if using spans for cols is valid so maybe it's perfectly fine.

Reply:
bothered me too, changed to divs.